core/src/modules/ops/export.ts

```diff
@@ -138,6 +138,33 @@ export namespace ExportOps {
     return _exports.length;
   }
 
+  /**
+   * Mark an array of Exports complete
+   */
+  export async function completeBatch(_exports: Export[]) {
+    if (_exports.length === 0) return;
+
+    const now = new Date();
+    await Export.update(
+      {
+        errorMessage: null,
+        errorLevel: null,
+        completedAt: now,
+        state: "complete",
+      },
+      {
+        where: { id: { [Op.in]: _exports.map((e) => e.id) } },
+        returning: true,
+      }
+    );
+
+    for (const _export of _exports) {
+      _export.state = "complete";
+      _export.completedAt = now;
+      await Export.logExport(_export);
```

Comment:
Is this doing the same thing twice? `logExport` is part of an `@AfterUpdate` on that model.

Reply:
`@AfterUpdate` only fires after a single-item update.  There's `@AfterBatchUpdate` which happens after these kind of bulk updates, but they don't actually include the rows modified (because they would often be too big to fit in RAM)